### DISCUSSION TITLE
High Voltage Alarm every time JKBMS starting a Cell OVP and Balance

### DISCUSSION THREAD

**Discussion by nolte44379** [Louisvdw/dbus-serialbattery#221] [Q&A] [answered]
Hi,

when the battery is almost full 99% and one Cell is count the Cell OVP Voltage 3.650V i get and ALARM on my Cerbo.

i can i deactivate the Alarm?

Settings in the JKBMS: Cell OVP 3.650
Cell OVPR 3.600
Start Balance 3.45

![Screenshot_20220929-174617](https://user-images.githubusercontent.com/113121410/193078241-8b639f8c-18f6-4b26-92da-6da6465effad.jpg)

![Screenshot_20220929-174604](https://user-images.githubusercontent.com/113121410/193078339-c5d5ab8e-fe87-4e23-9674-c76cbee70d2e.jpg)

![Screenshot_20220929-174017_VRM](https://user-images.githubusercontent.com/113121410/193078391-607c2021-99c9-439f-9e8e-5ad7b60954e9.jpg)

thanks martin

**Comment by Louisvdw** [comment, marked as answer]
If you get a high voltage alarm in your GX it means your battery and ESS system is now smart and working together. That is what you want. 
You need to fix your battery now, because it looks like you have cell imbalance issues.
See https://github.com/Louisvdw/dbus-serialbattery/wiki/FAQ#why-do-i-get-a-high-voltage-alarm
and https://github.com/Louisvdw/dbus-serialbattery/wiki/FAQ#why-is-the-max-cell-voltage-set-to-345v
